docs/openshift.md

```diff
@@ -0,0 +1,9 @@
+# OpenShift
+
+The place for the details on how to use the ExternalDNS operator in Red Hat OpenShift.
```

Comment:
Can be rephrased to "This document provides information about how to use the `ExternalDNS` Operator in OpenShift Container Platform. "